tests/test_scrubber.py

```diff
@@ -105,11 +105,9 @@ def test_breadcrumb_extra_scrubbing(sentry_init, capture_events):
         "password": "[Filtered]",
     }
 
-    assert event["_meta"] == {
-        "extra": {"auth": {"": {"rem": [["!config", "s"]]}}},
-        "breadcrumbs": {
-            "values": {"0": {"data": {"password": {"": {"rem": [["!config", "s"]]}}}}}
-        },
+    assert event["_meta"]["extra"]["auth"] == {"": {"rem": [["!config", "s"]]}}
+    assert event["_meta"]["breadcrumbs"] == {
+        "values": {"0": {"data": {"password": {"": {"rem": [["!config", "s"]]}}}}}
```

Comment:
Changed this because in py2.7 there's an additional `'sys.argv': {'': {'len': 11}}}}` in `event["_meta"]["extra"]`, I'm guessing from [here](https://github.com/getsentry/sentry-python/blob/master/sentry_sdk/integrations/argv.py). Unsure why this only happens on 2.7 but it doesn't seem to be something that needs to be fixed.